### PR TITLE
[#787] transfer country fields to jira

### PR DESCRIPTION
**Pull request by goreck888**
Add possibility to create tickets in jira with country of customer
and country of collaboration fields added to project.

### CODE REVIEW

File: app/models/country.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  def self.customer_countries_list
+    get_countries(NON_EUROPEAN)
+  end
+
+  def self.countries_list
+    get_countries(INTERNATIONAL, NON_EUROPEAN)
+  end
+
+  def self.validator_countries_list
```
Comment (bwilk) [inline app/models/country.rb]: I still think this should end up in the validator if it's called validator_countries_list. Contrary maybe changing the name would make up for this mismatch - maybe "allowed_countries" or similar would fit here. 

You don't need "_list" in this name no matter how you approach earlier comments. Ruby is always trying to be concise - let's respect this approach. Besides Ruby conventions this is not a list from the data type perspective - it's an array ;)

File: app/models/project.rb [hunk 2]
```diff
@@ -11,9 +12,20 @@ class Project < ApplicationRecord
             presence: true,
             uniqueness: { scope: :user, message: "Project name need to be unique" }
 
+  validates :country_of_customer, presence: true, inclusion: { in: Country.validator_countries_list }
```
Comment (bwilk) [inline app/models/project.rb]: could you unit-test all the validations in this model?

File: app/models/country.rb (hunk 1 above)
Comment (goreck888) [reply app/models/country.rb]: Ok, now it should be ok :) I changed the names of these arrays

File: app/models/project.rb (hunk 2 above)
Comment (goreck888) [reply app/models/project.rb]: Yes, of course, in this branch I'm focused on jira synchronization with new country fields and fix unit tests concerning this aspect of my task. This validator unit test will be written in the base branch 785-information-about-country, PR #872 and then 787 will be rebased and merged.

**Pull request closed without merging**